tests/api/test_product.py

```diff
@@ -740,7 +743,7 @@ def test_create_product(
     product_is_published = True
     product_charge_taxes = True
     product_tax_rate = "STANDARD"
-    product_price = 22.33
+    product_price = "22.33"
```

Comment:
Why did you change it to `str`? It should work with float

Reply:
@korycins these are money amounts and the `Product.price` is a `MoneyField` (`Decimal` beneath) that can accept string representations of decimal numbers. Floats aren't accurate and work here only because the amount is properly rounded while you cast it to string (the default precision is less than 17 decimal places):
```python
[ins] In [1]: f"{22.33:.17}"                                                                        
Out[1]: '22.329999999999998'
```

It's a good habit to pass strings to `Decimals` and generally use them instead of floats when one deals with money.